frontend/app/grave/[account]/page.tsx

```diff
@@ -18,37 +17,10 @@ export const metadata: Metadata = {
   },
 };
 
-const logoPath = '/images/logo-full.png';
-export default function Grave({ params }: { params: { account: string } }) {
+export default function GravePage({ params }: { params: { account: string } }) {
   return (
     <Container maxW={'6xl'} width={'100%'} py={5}>
-      <Stack
-        direction={{ base: 'column', md: 'row' }}
-        justify="space-between"
-        align="center"
-        w="100%"
-        pt="50px"
-      >
-        <GravePagePanel account={params.account} />
-        <Image
-          mx={'40px'}
-          src={logoPath}
-          alt="Universal-Grave-logo"
-          width={'300px'}
-        />
-      </Stack>
-      <Box>
-        <Text
-          fontSize="20px"
-          color="white"
-          fontFamily="Bungee"
-          mb="30px"
-          mt="30px"
-        >
-          GRAVEYARD
-        </Text>
-        <GravePageAssets account={params.account} />
-      </Box>
```

Comment:
maybe for the future, but having more components at the page level may make the page render faster (if there are non "use client" components) because they'll be rendered at the server 
If we put everything in one giant component that has "use client" we lose a lot of benfit